web/src/engine/websites/Dokuha.ts

```diff
@@ -0,0 +1,70 @@
+﻿import { Tags } from '../Tags';
+import icon from './Dokuha.webp';
+import { type Chapter, DecoratableMangaScraper, Page } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchJSON, FetchWindowScript } from '../platform/FetchProvider';
+import type { Priority } from '../taskpool/DeferredTask';
+
+type JSONImage = {
```

Comment:
From the implementation it seems image can be falsy, therefore it should be `undefined | { ... }`